src/tools/roc_conv/main.cpp

```diff
@@ -69,22 +70,26 @@ int main(int argc, char** argv) {
     pipeline::ConverterConfig config;
 
     if (args.frame_size_given) {
-        if (args.frame_size_arg <= 0) {
-            roc_log(LogError, "invalid --frame-size: should be > 0");
+        if (sizeof(args.frame_size_arg) <= 0) {
+            roc_log(LogError, "invalid --frame-size: frame size unspecified");
             return 1;
         }
```

Comment:
`sizeof(args.frame_size_arg)` will return `sizeof(char*)` i.e. size of the pointer, e.g. 8 on 64-bit machines. If you want to check that the string is non-empty, you should use strlen().

However there is no need to check it here because parse_duration() will do all necessary checks. We just need to check its return value.

Reply:
done.